src/main/java/fi/metatavu/metaform/server/rest/MetaformsApiImpl.java

```diff
@@ -296,11 +313,48 @@ public Response listReplies(UUID metaformId, UUID userId, String createdBeforePa
         modifiedAfter,
         includeRevisions != null && includeRevisions,
         fieldFilters);
-    
+    replies.forEach(reply -> auditLogEntryController.createAuditLogEntry(getLoggerUserId(), AuditLogEntryType.VIEW_REPLY,
```

Comment:
see comments above

Reply:
i would segragate listing for viewing, purpose if listing is usually not same as when viewing